build/pre-publish.js

```diff
@@ -272,8 +273,40 @@ async function transformDistributionFiles(rooltFolder, replacement) {
     }
 }
 
-function singleTransformZRRootFolder(code, replacement) {
-    return code.replace(/([\"\'])zrender\/src\//g, `$1zrender/${replacement}/`);
+/**
+ * 1. Transform zrender/src to zrender/lib
+ * 2. Add .js extensions
+ *
+ * FIXME. Comment in the named import statement is not supported.
```

Comment:
I tested it but it seems to be working?

Reply:
Ahhh I forget to remove this comment after this fix https://github.com/ecomfe/zrender/pull/863